Space-X Defense Program/Space-X Defense Program/CGPoint+VectorMath.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  CGPoint+VectorMath.swift
+//  Space-X Defense Program
+//
+//  Created by Su Haifeng on 3/19/18.
```

Comment:
If your just going to copy my code line for line do you really need to attribute it to yourself?

Reply:
Sorry about that~ when I created a new swift file ~ it automatically placed my name~ I will change this in the future~

Reply:
Check this out so you better understand. https://choosealicense.com/no-permission/